google/cloud/pubsub/samples/samples.cc

```diff
@@ -1388,9 +1369,17 @@ void AutoRunAvro(
   auto subscriber = google::cloud::pubsub::Subscriber(
       google::cloud::pubsub::MakeSubscriberConnection(subscription));
 
-  std::cout << "\nRunning CreateSubscription() sample [avro]" << std::endl;
-  CreateSubscription(subscription_admin_client,
-                     {project_id, avro_topic_id, subscription_id});
+  std::cout << "\nCreate test subscription (" << subscription.subscription_id()
+            << ") [avro]" << std::endl;
+  google::pubsub::v1::Subscription request;
+  request.set_name(subscription.FullName());
+  request.set_topic(topic.FullName());
+  subscription_admin.CreateSubscription(request);
```

Comment:
nit: I usually `(void)` these when I am not using the result.

Reply:
```suggestion
(void)  subscription_admin.CreateSubscription(request);
```